### DISCUSSION TITLE
Block inserter not showing with the last block when using ACF fields to the bottom of gutenburg editor

### DISCUSSION THREAD

**Discussion by mrhatia** [community/community#137172] [Projects and Issues]
### Body

Hi,

I am using the ACF plugin on my WordPress website to display some fields at the bottom of pages. However, I'm encountering difficulties with the editor experience—specifically, the block inserter and spacing do not show correctly with the last block in the editor when ACF is active. When I deactivate the ACF plugin, everything works fine.

Please see the attached screenshot below for reference.

![Block-inserter-not-showing](https://github.com/user-attachments/assets/9e50d6be-51d1-4363-8b27-7e0a11411494)

![block-inserter-showing](https://github.com/user-attachments/assets/611e269d-f133-4b41-8bb9-064e57c59e50)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by kathlene-cory** [comment]
Hi @mrhatia - Thanks for reaching out. 

Have you tried this on different screen sizes? Even adjusting the browser size may help. 

If the issue persists, I would recommend creating a child theme and adding this css code to slightly alter the admin view: 
`/* Increase the size of the block inserter */
.edit-post-header__toolbar .block-editor-inserter__toggle {
    font-size: 2rem; /* Adjust size as needed */
    padding: 10px;   /* Adjust padding for more space */
}

/* Always show the block inserter button */
.block-editor-inserter__toggle {
    display: block !important;
}
`

You will need admin access to create a child theme. Hope this helps. 

Happy coding!
